app/models/domain.rb

```diff
@@ -0,0 +1,12 @@
+class Domain < ApplicationRecord
+  self.table_name = 'domains'
+
+  has_many :email_redirects, class_name: 'Email::Redirect', foreign_key: :domain_id, inverse_of: :domain, dependent: :restrict_with_error
+
+  validates :fqdn, presence: true
+  validates :fqdn, uniqueness: true
+
+  def display_name
+    fqdn.to_s
```

Comment:
@dmitry-sinina you don't need to add `.to_s` explicitly because you've already have a valiation that value can't be nil